client/client.go

```diff
@@ -0,0 +1,203 @@
+package client
+
+import (
+	"database/sql"
+	_ "github.com/mattn/go-sqlite3"
+)
+
+type Client struct {
+	client *sql.DB
+}
+
+// My Name
+type Name struct {
+	Id    int    `json:"id"`
+	First string `json:"firstname"`
+	Last  string `json:"lastname"`
+}
+
+type User struct {
+	Id       int    `json:"id"`
+	Username string `json:"first"`
+	Password string `json:"password"`
+	Admin    int    `json:"admin"`
+}
+
+func New() *Client {
+
+	c, err := sql.Open("sqlite3", "./foo.db")
+	if err != nil {
+		panic(err)
+	}
+
+	conn := &Client{c}
+
+	return conn
+}
+
+func init() {
+
+	db := New()
+
+	CreateTables(db)
+
+	myName := &Name{Id: 1, First: "Happy", Last: "Gilmore"}
+	user := &User{1, "jbot", "secretJB", 1}
+	userStandard := &User{2, "regs", "password", 0}
+
+	WriteName(db, myName)
+	AddUser(db, user)
+	AddUser(db, userStandard)
+
+	db.client.Close()
+
+}
+
+func CreateTables(db *Client) {
```

Comment:
We should investigate an ORM, not within the scope of this PR though.

Reply:
You're writing all the sql and data layer by hand? That's the boilerplate I was hoping to avoid with the swagger client gen.